terminus/geometry/latlon.py

```diff
@@ -64,55 +65,71 @@ def __repr__(self):
     def sum(self, other):
         return LatLon(self.lat + other.lat, self.lon + other.lon)
 
+    def curvature(self):
+        # Radius of planet curvature (meters)
+        curvature = Ecef.equatorial_radius() / math.sqrt(1 - Ecef.first_eccentricity_parameter() ** 2 * self._sin_lat() ** 2)
+        return curvature
+
+    def latlon_to_ecef(self):
```

Comment:
You are already on the LatLon class, no need to append that to the method name. It should just be `to_ecef`.